### PR TITLE
[META] Update CODEOWNERS

### PR DESCRIPTION
**Pull request by skovati**
* **Tickets addressed:** N/A
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

Switches the CODEOWNERS to @NASA-AMMOS/aerie-backend, so we can leverage the built-in github code review goodies. The backend team is currently set up to automatically request 2 members as reviewers with a load balancing algorithm. It is recommended to manually request one or two contributors you think would be a good fit to review your PR.

### CODE REVIEW

**Comment by skovati** [comment]
As you can see above, by requesting review from the aerie-backend team, Github automagically requested review from pcrosemurgy and Mythicaeda. Of course, anyone from the aerie-backend team is still able to approve this PR, which would count towards the CODEOWNER required approvals

**Review by camargo — APPROVED** [review]

**Review by Mythicaeda — APPROVED** [review]

**Pull request merged**
